host-ext/crouton/window.js

```diff
@@ -262,8 +269,10 @@ function handleFocusBlur(evt) {
 }
 
 /* Start in full screen */
-chrome.windows.update(chrome.windows.WINDOW_ID_CURRENT,
-                      {'state': "fullscreen"}, function(win) {})
+if (location.search.search(/[&?]mode=f(&|$)/i) != -1) {
```

Comment:
Yeah, I'll do that.

Also, I just read [[http://speakingjs.com/es5/ch16.html]]...I suspect we have a lot of unintended global variables in the code...